go.mod

```diff
@@ -0,0 +1,25 @@
+module github.com/giantswarm/net-exporter
+
+go 1.14
+
+require (
+	github.com/giantswarm/exporterkit v0.2.0
+	github.com/giantswarm/k8sclient v0.2.0
+	github.com/giantswarm/microerror v0.2.0
+	github.com/giantswarm/micrologger v0.3.1
+	github.com/gogo/protobuf v1.3.1 // indirect
+	github.com/google/go-cmp v0.4.0
+	github.com/google/gofuzz v1.1.0 // indirect
+	github.com/imdario/mergo v0.3.8 // indirect
+	github.com/miekg/dns v1.1.29
+	github.com/pelletier/go-toml v1.6.0 // indirect
+	github.com/prometheus/client_golang v1.6.0
+	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073 // indirect
+	golang.org/x/net v0.0.0-20200301022130-244492dfa37a // indirect
+	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
+	k8s.io/api v0.17.2
```

Comment:
Following from the discussion in ingress-exporter, I think this should be v0.16.6.

Reply:
Why do we use 0.17.2 in aws-operator then?

Reply:
Because `aws-operator` overrides it with `replace` directive. And why does it do that? It's something we have found during hackathon.

Problem: https://gigantic.slack.com/archives/CUR85UBPC/p1585131916023600?thread_ts=1585129348.018500&cid=CUR85UBPC

Solution: https://gigantic.slack.com/archives/CUR85UBPC/p1585133058024800?thread_ts=1585129348.018500&cid=CUR85UBPC

External link containing solution: https://github.com/kubernetes/kubernetes/issues/79384#issuecomment-505627280

**Except** we use replace with versions for `1.16.6` instead of `1.15`

Reply:
Essentially, the version you indicate here does not matter. The final version used is the one indicated in `replace` block.

Reply:
Thanks. I'd rather use v0.0.0 there then to avoid confusion but that's unrelated.

Can I get my approval pls?